bucket/syncthing.json

```diff
@@ -14,7 +15,20 @@
             "extract_dir": "syncthing-windows-386-v1.2.0"
         }
     },
-    "bin": "syncthing.exe",
+    "bin": [
+        [
+            "syncthing.exe",
+            "syncthing",
+            "-home \"$dir\\config\""
+        ]
+    ],
+    "persist": "config",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\config\") -and (Test-Path \"$Env:LocalAppdata\\Syncthing\")) {",
+        "    New-Item \"$dir\\config\" -ItemType 'Directory' -Force | Out-Null",
+        "    Copy-Item \"$Env:LocalAppdata\\Syncthing\\*\" \"$dir\\config\" -Exclude LOCK -Recurse -Force",
```

Comment:
Should this be Move-Item or Cooy-Item?

Reply:
I prefer to copy them because I don't want to break other thing. If users install syncthing and uninstall -p , they won't lose anything.